docs/recipes/WritingTests.md

```diff
@@ -311,46 +303,41 @@ can be tested like:
 
 ```js
 import React from 'react'
-import Enzyme, { shallow } from 'enzyme'
-import Adapter from 'enzyme-adapter-react-16'
+import { render, fireEvent, getByTestId } from '@testing-library/react'
 import Header from '../../components/Header'
 
-Enzyme.configure({ adapter: new Adapter() })
-
-function setup() {
-  const props = {
-    addTodo: jest.fn()
-  }
-
-  const enzymeWrapper = shallow(<Header {...props} />)
-
-  return {
-    props,
-    enzymeWrapper
-  }
-}
-
 describe('components', () => {
   describe('Header', () => {
```

Comment:
Personally I think having these `describe` blocks are useless.

- `components`: yes, I'm obviously testing components in this file
- `Header`: yes, this is the test file for the header, I'm obviously testing the Header component.

I'd recommend ditching `describe` in favor of simply `test`: https://kentcdodds.com/blog/avoid-nesting-when-youre-testing

Reply:
btw, thought of changing the `it`s to `test`s but It meant changing the whole document and I'm not sure everyone's onboard